FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v03/run_digi_reco.py

```diff
@@ -55,9 +55,9 @@
 #
 inputfile = "ALLEGRO_sim.root"  # input file produced with ddsim
 Nevts = -1                      # -1 means all events
-addNoise = False                # add noise or not to the cell energy
+addNoise = True                # add noise or not to the cell energy
```

Comment:
Till filtering works, I would set addNoise to False by default - to avoid producing huge output files

Reply:
addNoise is switched off by default.